recipe/meta.yaml

```diff
@@ -1,88 +1,108 @@
+    - pkg-config
     - libtool  # [not win]
-    - nasm  # [(osx and x86_64) or linux32 or linux64]
+    - nasm  # [win or (osx and x86_64) or (linux and (not s390x))]
     - make  # [not win]
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
-    - bzip2  # [not win]
-    - freetype  # [not win]
-    - gnutls  # [not win]
-    - libiconv  # [not win and not linux]
-    - x264  # [not win]
-    - zlib  # [not win]
+    - aom
+    - bzip2
+    - dav1d
+    - freetype
+    - fontconfig
+    - harfbuzz
+    - libiconv
+    - zlib
     - openh264  # [not win]
+    - openjpeg >=2.1.0   # [not win]
+    - librsvg  # [osx]
+    - libtheora
+    - libvorbis # [not win]
+    - libxml2
+    - tesseract  # [not (win or s390x)]
     - lame  # [not win]
-    - gmp  # [unix]
-    - libvpx  # [not win]
+    - libvpx  # [not (win or s390x)]
     - libopus # [not win]
-    - openssl # [not win]
+    - openssl
   run:
     - lame  # [not win]
+    - xz
+    - libglib  # [osx]
+    - cairo  # [not win]
+    - libiconv
 
 test:
   commands:
     - ffmpeg --help
-    - ffmpeg -loglevel panic -codecs
     - ffmpeg -loglevel panic -protocols | grep "https"  # [not win]
     - ffmpeg -loglevel panic -codecs | grep "libmp3lame"  # [not win]
-    - ffmpeg -loglevel panic -codecs | grep "DEVI.S zlib"  # [unix]
-    - ffmpeg -loglevel panic -codecs | grep "DEV.LS h264"  # [linux64 or osx64]
-    - ffmpeg -loglevel panic -codecs | grep "D.V.LS h264"  # [ppc64le]
-    - ffmpeg -loglevel panic -codecs | grep "libx264"  # [linux64 or osx64]
-    - ffmpeg -loglevel panic -codecs | grep "libopenh264"  # [linux64 or osx64]
+    - ffmpeg -loglevel panic -codecs | grep "DEVI.S zlib"  # [not win]
+    - ffmpeg -loglevel panic -codecs | grep "DEV.LS h264"  # [not (win or s390x)]
+    - ffmpeg -loglevel panic -codecs | grep "libopenh264"  # [not win]
     # Verify dynamic libraries on all systems
     {% set ffmpeg_libs = [
         "avcodec",
         "avdevice",
         "swresample",
-        "postproc",
         "avfilter",
-        "swresample",
         "avcodec",
         "avformat",
-        "swscale",
-        "avresample"
+        "swscale"
     ] %}
     {% for each_ffmpeg_lib in ffmpeg_libs %}
-    - test -f $PREFIX/lib/lib{{ each_ffmpeg_lib }}.dylib  # [osx]
-    - test -f $PREFIX/lib/lib{{ each_ffmpeg_lib }}.so     # [linux]
+    - test -f $CONDA_PREFIX/lib/lib{{ each_ffmpeg_lib }}.dylib  # [osx]
+    - test -f $CONDA_PREFIX/lib/lib{{ each_ffmpeg_lib }}.so     # [linux]
+    - if not exist %PREFIX%\Library\bin\{{ each_ffmpeg_lib }}*.dll exit 1  # [win]
+    - if not exist %PREFIX%\Library\bin\{{ each_ffmpeg_lib }}.lib exit 1  # [win]
     {% endfor %}
 
 about:
-  home: http://www.ffmpeg.org/
-  license: GPL 3
-  license_file: COPYING.GPLv3  # [unix]
+  home: https://www.ffmpeg.org
+  license: GPL-2.0-or-later AND GPL-3.0-or-later AND LGPL-3.0-or-later
+  license_file:
+    - COPYING.GPLv2
+    - COPYING.GPLv3
+    - COPYING.LGPLv2.1
```

Comment:
If we use `--enable-version3`, should we only have `COPYING.LGPLv3` in here?

Reply:
done